launchable/commands/record/session.py

```diff
@@ -149,3 +181,49 @@ def session(
             raise e
         else:
             click.echo(e, err=True)
+
+    if session_name:
+        try:
+            add_session_name(
+                client=client,
+                build_name=build_name,
+                session_id=session_id,
+                session_name=session_name,
+            )
+        except Exception as e:
+            if os.getenv(REPORT_ERROR_KEY):
+                raise e
+            else:
+                click.echo(e, err=True)
+
+
+def add_session_name(
+    client: LaunchableClient,
+    build_name: str,
+    session_id: str,
+    session_name: str,
+):
+    sub_path = "builds/{}/test_sessions/{}".format(build_name, session_id)
+    payload = {
+        "name": session_name
+    }
+    res = client.request("patch", sub_path, payload=payload)
+
+    if res.status_code == HTTPStatus.NOT_FOUND:
+        click.echo(
+            click.style(
+                "Test session {} was not found. Record session may be failed.".format(session_id),
+                'yellow'),
+            err=True,
+        )
+        sys.exit(1)
+    if res.status_code == HTTPStatus.BAD_REQUEST:
+        click.echo(
+            click.style(
+                "You cannot use test session name {} since it is already used by other test session in your workspace."
```

Comment:
[nits]
At this point, the `record session` should be done correctly, and it is failing on adding name. I think it is better to advise the test session is created.
`"You cannot use test session name {} since it is already used by other test session in your workspace. The record session is completed successfully without session name."`